cmd/clusterctl/client/move_test.go

```diff
@@ -80,6 +80,49 @@ func Test_clusterctlClient_Move(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "returns an error if both move ToDirectory and FromDirectory is set",
+			fields: fields{
+				client: fakeClientForMove(),
+			},
+			args: args{
+				options: MoveOptions{
+					ToDirectory:   "/var/cache/toDirectory",
+					FromDirectory: "/var/cache/fromDirectory",
+				},
+			},
+			wantErr: true,
+		},
+		{
+			name: "returns an error if neither FromDirectory, ToDirectory, or ToKubeconfig is set",
```

Comment:
There is some nuance to be considered here. 
The test here only works becasue it returns an error at the `getClusterClient` function call and this is becasue the test setup is not configured to find a clusterclient for emtpy kubeconfig. However, in real world this will not return an error as the `getClusterClient` will use default discovery rules when kubeconfig is not specified.

We have validation for `clusterctl move` at the `cmd` level to make sure that atleast one target (from-directory, to-directory, to-kubeconfig) is specified to but we dont have valiation to make sure that atleast one input is specific at the `client` level. Given that clusterctl is also used as a library I would suggest to add something like this at l.82 of `move.go`: 
```go
if options.FromDirectory == "" && options.ToDirectory == "" && options.ToKubeconfig == (Kubeconfig{}) {
	return errors.Errorf("at least one of FromDirectory, ToDirectory and ToKubeconfig must be provided")
}
```
(make adjustments needed to work with `DryRun` as needed.)